examples/prometheus/prometheus.yaml

```diff
@@ -724,6 +724,30 @@ objects:
           action: keep
           regex: router;1936-tcp
 
+      # Scrape config for Service Catalog controllers
+      #
+      # Catalog runs on each master node and exposes a /metrics endpoint on :6443 that contains operational metrics for
+      # the controllers.
+      #
+      - job_name: 'catalog-controllers'
+
+        scheme: https
+        tls_config:
+          server_name: 'controller-manager.kube-service-catalog.svc'
+          ca_file: /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+        bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+
+        kubernetes_sd_configs:
+        - role: endpoints
+          namespaces:
+            names:
+            - kube-service-catalog
+
+        relabel_configs:
+        - source_labels: [__meta_kubernetes_service_name]
+          action: keep
+          regex: controller-manager
```

Comment:
i think @smarterclayton had some concerns about more stuff being added in here, so need him to approve.

Reply:
If you don't expose tenant metrics you just want to set the annotations on your service and be in the generic block up above.